design-doc.md

```diff
@@ -0,0 +1,179 @@
+## Table of Contents
+
+
+{{TOC}}
+
+
+
+## Links to Old Documents {#links-to-old-documents}
+
+
+
+*   [https://docs.google.com/document/d/1b7Or5gidpfQenkGR4dysz9ShEWgrWxxPJuQUBrnRXvs/edit#heading=h.3svxzuabblb4](https://docs.google.com/document/d/1b7Or5gidpfQenkGR4dysz9ShEWgrWxxPJuQUBrnRXvs/edit#heading=h.3svxzuabblb4)
+*   [https://docs.google.com/document/d/1smOrLXBs4E9KvRcjFzVFSGRLSOdJFioi8HHv_xvMlCI/edit](https://docs.google.com/document/d/1smOrLXBs4E9KvRcjFzVFSGRLSOdJFioi8HHv_xvMlCI/edit)
+*   [https://docs.google.com/document/d/1hFMPtjQnlscUwIauxcjAPxFmRAgruZ-6TBnKGEW4PWc/edit#heading=h.3svxzuabblb4](https://docs.google.com/document/d/1hFMPtjQnlscUwIauxcjAPxFmRAgruZ-6TBnKGEW4PWc/edit#heading=h.3svxzuabblb4)
+*   [https://docs.google.com/document/d/1-V07kO9L1mab8GV1-OQoLTAIDPZyWi5P_PRoIsEHeik/edit#heading=h.3svxzuabblb4](https://docs.google.com/document/d/1-V07kO9L1mab8GV1-OQoLTAIDPZyWi5P_PRoIsEHeik/edit#heading=h.3svxzuabblb4)
+*   [https://docs.google.com/document/d/1fiRlWzmeHYXGtMXEUCcs82sLQDyZc1oSdY5lUs566jc/edit](https://docs.google.com/document/d/1fiRlWzmeHYXGtMXEUCcs82sLQDyZc1oSdY5lUs566jc/edit)
+*   [https://docs.google.com/document/d/1p8IW_QuzMO6EhLiBOPJY0FqD0ZoS64-7M_aZvP9y2-8/edit#](https://docs.google.com/document/d/1p8IW_QuzMO6EhLiBOPJY0FqD0ZoS64-7M_aZvP9y2-8/edit#)
+*   [https://github.com/dcos-labs/ui-kit/blob/master/CONTRIBUTING.md](https://github.com/dcos-labs/ui-kit/blob/master/CONTRIBUTING.md)
+*   [https://docs.google.com/document/d/1p-CnUnhNFVz6WomPQvMEsTybbGgV0FHWgsUzBqX64cs/edit#heading=h.apvnjfls8tkq](https://docs.google.com/document/d/1p-CnUnhNFVz6WomPQvMEsTybbGgV0FHWgsUzBqX64cs/edit#heading=h.apvnjfls8tkq)
+
+
+## Living Document
+
+This is a living document[^1]. Feel free to add or change information. Everybody should be able to contribute to this document.
+
+
+## General Guidelines
+
+
+### Why are we building the design system?
+
+Have one source of truth, build a consistent product across company
```

Comment:
Isn't present continuous tense better here?
 `Having one source of truth, building a consistent product across company and platforms, increasing developer productivity and feature delivery.`